README.md

```diff
@@ -56,26 +54,21 @@ This is done by adding the block below for the respective plugin to the file [we
    "path": "scaffolding",
    "repo": "hashicorp/packer-plugin-scaffolding",
    "branch": "main",
-   "artifactDir": ".doc-artifacts"
+   "version": "latest"
  }
 ```
 
+If a plugin maintainer wishes to only include a specific version of released docs then the `"version"` key in the above configuration should be set to a released version of the plugin.
 
-The documentation structure needed for Packer.io can be generated manually, but it is
-encouraged to use the action on release events so that documentation stays up to date.
 
-Requirements:  Nodejs and NPM
+The documentation structure needed for Packer.io can be generated manually, by creating a simple zip file called `docs.zip` of the docs directory and included in the plugin release.
 
-In the plugin root execute the command packer-docs-artifact command
-
-```
-> npx -p @hashicorp/packer-docs-artifacts generate
+```/bin/bash
+[[ -d docs/ ]] && zip -r docs.zip docs/
 ```
 
-The generated files will be placed under `PLUGIN_ROOT/.doc-artifacts`; this directory contains all the docs
-and respective navigation information needed for including the plugin docs under packer.io/docs/
+Once the first `docs.zip` file has been included into a release you will need to open a one time pull-request against [hashicorp/packer](https://github.com/hashicorp/packer) to register the plugin docs.
 
-A one time pull-request will needs to be opened against [hashicorp/packer](https://github.com/hashicorp/packer) to register the plugin docs.
 This is done by adding the block below for the respective plugin to the file [website/data/docs-remote-navigation.js](https://github.com/hashicorp/packer/blob/master/website/data/docs-remote-plugins.json).
```

Comment:
I think this section, from "This is done..." to the start of the  "Requirements" section, might be duplicated above. So seems we can cut it 🔪